include/mapnik/geometry_is_valid.hpp

```diff
@@ -106,6 +110,140 @@ inline bool is_valid(T const& geom)
     return detail::geometry_is_valid() (geom);
 }
 
+typedef boost::geometry::model::d2::point_xy<double> Point;
```

Comment:
We already have defined Point in `include/mapnik/geometry.hpp` that works with boost algorithms, we should probably use that if we need Point declared.